com.github.paulwedeck.settlers-remake.yaml

```diff
@@ -0,0 +1,48 @@
+app-id: com.github.paulwedeck.settlers-remake
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: jsettlers
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --device=dri
+  - --share=network
+  - --socket=pulseaudio
+
+modules: 
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk11/install.sh
+
+  - name: jsettlers
+    buildsystem: simple
+    build-commands:
+      - cp -r JSettlers $FLATPAK_DEST/JSettlers
+      - install -Dm755 jsettlers.sh $FLATPAK_DEST/bin/jsettlers
+      - install -Dm755 jsettlers-mapcreator.sh $FLATPAK_DEST/bin/jsettlers-mapcreator
+      - install -Dm644 icon.png  $FLATPAK_DEST/share/icons/hicolor/128x128/apps/$FLATPAK_ID.png
+      - install -Dm644 com.github.paulwedeck.settlers-remake.desktop -t  $FLATPAK_DEST/share/applications
+      - install -Dm644 com.github.paulwedeck.settlers-remake.mapcreator.desktop -t  $FLATPAK_DEST/share/applications
+      - install -Dm644 com.github.paulwedeck.settlers-remake.metainfo.xml -t  $FLATPAK_DEST/share/metainfo
+    sources:
+      - type: archive
+        url: https://github.com/paulwedeck/settlers-remake/releases/download/v0.6.0.3-alpha/JSettlers.zip
+        sha256: 9ba7b0e0ce139fe73ca25b711f92b1e9b365997c6ca8ce5ee06fb0f9c1aff67e
+        dest: JSettlers
+      - type: file
+        url: https://github.com/paulwedeck/settlers-remake/raw/5d15b723153d9ac1af3d2d8b7ae25911162d84f1/jsettlers.main.swing/src/main/resources/jsettlers/main/swing/icon.png
+        sha256: 830aa4abeff112a3ca8002fcab0677cf5329e3575687c8e4bce8979130824ba7
+      - type: file
+        path: com.github.paulwedeck.settlers-remake.desktop
+      - type: file
+        path: com.github.paulwedeck.settlers-remake.mapcreator.desktop
+      - type: file
+        path: com.github.paulwedeck.settlers-remake.metainfo.xml
```

Comment:
these 4 files are in the .zip

```
     2730  10-02-2022 16:57   JSettlers/icon.png
        0  10-02-2022 16:58   JSettlers/dist/
        0  10-02-2022 16:57   JSettlers/dist/linux/
     5383  10-02-2022 16:57   JSettlers/dist/linux/com.github.paulwedeck.settlers-remake.metainfo.xml
      363  10-02-2022 16:57   JSettlers/dist/linux/com.github.paulwedeck.settlers-remake.mapcreator.desktop
      410  10-02-2022 16:57   JSettlers/dist/linux/com.github.paulwedeck.settlers-remake.desktop
```

So they should be installed from there.

Reply:
Yes, they are currently in the zip file but this won't be the case starting with the new release. There is no need for these files in the main project or release file because they use flatpak specific paths.

Reply:
**they are not flatpak specific at all.**

a UI app without icon or .deskop is a bug

the appstream file is also standard now.